WordPressCom-Stats-iOS/StatsTableViewController.m

```diff
@@ -1345,5 +1349,27 @@ - (void)resetDateToTodayForSite
     self.selectedDate = date;
 }
 
+- (void)trackViewControllerAnalytics
+{
+    WPAnalyticsStat stat;
+    
+    switch (self.selectedPeriodUnit) {
+        case StatsPeriodUnitDay:
+            stat = WPAnalyticsStatStatsPeriodDaysAccessed;
+            break;
+        case StatsPeriodUnitWeek:
+            stat = WPAnalyticsStatStatsPeriodWeeksAccessed;
+            break;
+        case StatsPeriodUnitMonth:
+            stat = WPAnalyticsStatStatsPeriodMonthsAccessed;
+            break;
+        case StatsPeriodUnitYear:
+            stat = WPAnalyticsStatStatsPeriodYearsAccessed;
+            break;
+    }
+    
+    [WPAnalytics track:stat];
```

Comment:
There shouldn't ever be an unknown state - and without providing a default case then there will be a compiler warning if we add another enum value.